versions/dependencies.js

```diff
@@ -112,7 +112,7 @@ module.exports = {
 	'react-stub-context': '^0.7.0',
 	'regenerator-runtime': '^0.13.1',
 	rimraf: '^2.6.2',
-	'sass-lint': '1.13.0',
+	'sass-lint': '1.13.1',
```

Comment:
Maybe we could make it this way ?
```suggestion
	'sass-lint': '^1.13.1',
```